debian/ideascube.postinst

```diff
@@ -31,9 +31,6 @@ case "$1" in
         ln -fs /etc/nginx/sites-available/$DPKG_MAINTSCRIPT_PACKAGE /etc/nginx/sites-enabled/$DPKG_MAINTSCRIPT_PACKAGE
         echo "Linking uWSGI app"
         ln -fs /etc/uwsgi/apps-available/$DPKG_MAINTSCRIPT_PACKAGE.ini /etc/uwsgi/apps-enabled/$DPKG_MAINTSCRIPT_PACKAGE.ini
-        echo "Restarting Nginx and uWSGI"
-        systemctl restart nginx.service uwsgi.service
-        systemctl enable uwsgi.service
```

Comment:
But I don't think it should.

You should **reload** nginx and uwsgi, there's no reason to **restart** them.

And it should only be needed on first install, I'm pretty sure subsequent installs don't need it. (see the `touch-reload` I'm adding to the uwsgi config file in another commit)